Sources/OutputStream+Write.swift

```diff
@@ -0,0 +1,72 @@
+//
+//  OutputStream+Write.swift
+//  FTAPIKit
+//
+//  Created by Matěj Kašpar Jirásek on 27/02/2019.
+//  Copyright © 2019 FUNTASTY Digital s.r.o. All rights reserved.
+//
+
+import Foundation
+
+extension OutputStream {
+    private static let streamBufferSize = 1024
```

Comment:
[Speculation:] IMHO 1K buffer is too short. I’ve searched internet and found out, 4K is considered as “moderate” choice, since it does not pressures on CPU cache and also does not overlow size of page on most of the systems. I could not find any hard proof however.

Reply:
The guidance is here:

https://developer.apple.com/library/archive/documentation/Cocoa/Conceptual/Streams/Articles/ReadingInputStreams.html

> The best approach is to use some reasonable buffer size, such as 512 bytes, one kilobyte (as in the example above), or a page size (four kilobytes).

I agreee 4 KB might be more effective on current devices.